#165954089 chore(setup): setup development

TamunoibiAprekuma opened the pull request:
This PR sets up testing environment

[finishes #165954089]

Commit:
chore(setup): setup development

This PR setsup testing environment

[finishes #165954089]

Akinmyde commented on README.md:
```diff
@@ -1 +1,6 @@
+# Banka
+[![Build Status](https://travis-ci.com/TamunoibiAprekuma/Banka-1.svg?branch=develop)](https://travis-ci.com/TamunoibiAprekuma/Banka-1)
+
+[![Coverage Status](https://coveralls.io/repos/github/TamunoibiAprekuma/Banka/badge.svg?branch=master)](https://coveralls.io/github/TamunoibiAprekuma/Banka?branch=master)
```
Why is the branch set to master

TamunoibiAprekuma replied:
I do not really understand the question. I should make the branch develop?

Akinmyde commented on README.md:
```diff
@@ -1 +1,6 @@
-# Banka
\ No newline at end of file
+# Banka
+[![Build Status](https://travis-ci.com/TamunoibiAprekuma/Banka-1.svg?branch=develop)](https://travis-ci.com/TamunoibiAprekuma/Banka-1)
```
Why is the branch set to develop

TamunoibiAprekuma replied:
I do not really understand the question

Commit:
On branch ch-setup-environments-165954089
 Changes to be committed:
	renamed:    .babel.rc -> .babelrc
	modified:   .gitignore
	modified:   README.md
	deleted:    app.js
	modified:   package-lock.json
	modified:   package.json

Commit:
new file:   server/test/index.test.js

Commit:
Chore(Serup): setup environment

This pull request setsup
    - Test Environment
    - Development Environment

	modified:   package.json
	new file:   server/app.js
	new file:   server/routes/index.js
	modified:   server/test/index.test.js

[finishes #165954089]
